resources/cdk/photo_asset_manager/SPECIFICATION.md

```diff
@@ -1,130 +1,128 @@
 # Photo Asset Management Tech Spec
 
-This is the Tech Spec for the Photo Asset Management cross services example. This doc details the specifics of the deliverable app.
+This is the tech spec for the Photo Asset Management cross-service example. This doc details the specifics of the deliverable app.
 
-## Reading this document
+## How to read this document
 
-This document is broken down into the following sections
+This contains the following sections
```

Comment:
redundant?
```suggestion
```

Reply:
I think I originally intended this section to be more explanatory and less Table of Contentsy. I'll change to Table of contents and remove this line.